packages/adapter-node/src/index.js

```diff
@@ -2,9 +2,9 @@ import { createServer } from './server';
 /*eslint import/no-unresolved: [2, { ignore: ['\.\/app\.js$'] }]*/
 import * as app from './app.js';
 
-const { PORT = 3000 } = process.env; // TODO configure via svelte.config.js
+const { HOST = '0.0.0.0', PORT = 3000 } = process.env; // TODO configure via svelte.config.js
```

Comment:
Is this TODO still a TODO? Would the env vars take precedence?

Reply:
it's still a TODO insofar as you can't specify a default port/host via the adapter config. Is that something worth having, if we have environment variables?

Reply:
Well yeah what I meant is: is this something we still want to do? I don't have especially strong opinions about that. I assume it would be additional args you pass to the adapter factory function and not somewhere else in the config.

Reply:
it would be, yeah. tbh i'm inclined to just remove the TODO comment — it's less confusing if there's only one way to set non-default values, and we need environment variables regardless so that it's configurable at runtime